lib/route/mdb.c

```diff
@@ -0,0 +1,289 @@
+/*
+ * lib/route/mdb.c		Multicast Database
```

Comment:
for new files, please add a `/* SPDX-License-Identifier: LGPL-2.1-only */` line as the first line, and don't add the redundant license text comments.

The copyright comments only apply if you copied and adjusted the code. I guess, you did actually take the existing code and adjusted it. In that case, it's correct to preserve the copyright comments.